packages/i18n/locales/en/translation.json

```diff
@@ -1,4 +1,25 @@
 {
+  "actionCategory": {
+    "authorizationsDescription": "Grant and revoke authorizations, or perform an authorized execution.",
+    "authorizationsLabel": "Authorizations",
+    "chainGovernanceDescription": "Interact with the governance of the blockchain.",
+    "chainGovernanceLabel": "Chain Governance",
+    "customDescription": "Execute any custom message.",
```

Comment:
> Execute custom actions.

Let's remove the word "messages." It's an implementation detail from most users' perspective. The reason you and I are talking here is because we want to abstract messages away *using* Actions.

Reply:
soooo true